docs/releasing/publishing-a-preview.md

```diff
@@ -0,0 +1,46 @@
+# Publish preview of GOV.UK Frontend
+
+This preview guidance is aimed at Design System team members. If you're an external contributor who needs to create a preview, please [contact the Design System team](https://design-system.service.gov.uk/get-in-touch/) and we'll do it for you.
+
+Before you publish a preview, you need to have committed a code change to GOV.UK Frontend. Then follow these instructions.
+
+Use previews when you:
+
+- [work on developing a component or pattern](https://design-system.service.gov.uk/community/develop-a-component-or-pattern/) for the GOV.UK Design System
+- want to trial an experimental feature (guidance on trialing experimental features is in development)
+
+:warning: Your projects should never depend on a preview GOV.UK Frontend package. This is because someone could remove the GitHub branch containing the preview package at any time. For this reason, never use a preview package in a production setting.
```

Comment:
**suggestion** Just taking the opportunity to turn that to [a warning block](https://github.com/orgs/community/discussions/16925) with more impact than an emoji when rendered on Github:

```suggestion
> **Warning** Your projects should never depend on a preview GOV.UK Frontend package. This is because someone could remove the GitHub branch containing the preview package at any time. For this reason, never use a preview package in a production setting.
```